### DISCUSSION TITLE
Delayed Hydration

### DISCUSSION THREAD

**Discussion by kilobyte2007** [nuxt/nuxt#16354] [Questions] [answered]
Hey guys, congrats on the release! We're releasing our new site on top of it pretty soon!

I remember previously in some thread there were talks that delayed hydration will be available out of the box in future versions of Nuxt but it seems it never landed. Is there a plan to make it possible to mark specific components to never have hydration?

For example, our blog has purely text so we don't need hydration for it, we don't need the JS that renders it (which loads the whole markdown library) as well as the text itself which is loaded twice on hydration.

Thanks!

**Comment by Amirali-Yavari** [comment]
hi you can make your component `client only` this handle the hydration

**Comment by TheAlexLichter** [comment, marked as answer]
Wait for https://github.com/nuxt/framework/pull/5688

**Reply by kilobyte2007** [reply]
Thanks a lot, didn't know there was an open issue for that!
